docs/installingNative.md

```diff
@@ -181,48 +193,76 @@ Most of these prerequisites can be installed using the package manager of your O
 
   You may need to restart the PostgreSQL server after changing the file above.
 
-- Run the test suite (Docker must be installed and running):
-
-  ```sh
-  make test
-  ```
+## Running
 
-- Run the linters:
+If you have your own [PrairieLearn course repository](./requestCourse/index.md), you will need to create the file `PrairieLearn/config.json` with the path of your local course repository. Additionally, if you need support for workspaces, you should provide a path to a directory into which temporary files will be saved when using the in-browser file editor. Here is a sample configuration:
```

Comment:
"you should provide a path to a directory into which temporary files will be saved when using the in-browser file editor" is a non sequitur from "if you need support for workspaces".

Reply:
`Additionally` was removed.

Reply:
Changed in da99c72.